robot_0:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: position_controllers/JointPositionController
    joint: yaw_joint_l1
  joint2_position_controller:
    type: position_controllers/JointPositionController
    joint: yaw_joint_l2
  joint3_position_controller:
    type: position_controllers/JointPositionController
    joint: yaw_joint_r1
  joint4_position_controller:
    type: position_controllers/JointPositionController
    joint: yaw_joint_r2


  # Velocity Controllers ---------------------------------------
  joint5_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: drive_joint_l1
  joint6_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: drive_joint_l2
  joint7_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: drive_joint_r1
  joint8_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: drive_joint_r2
